RadarSDK/RadarLocationManager.m

```diff
@@ -875,7 +875,6 @@ - (void)sendLocation:(CLLocation *)location stopped:(BOOL)stopped source:(RadarL
                 self.sending = NO;
                 
                 [self updateTrackingFromMeta:config.meta];
-                [RadarSettings setFeatureSettings:config.meta.featureSettings];
```

Comment:
I think we want to update the sdk configuration here:
``` 
[RadarSettings setSdkConfiguration:config.meta.sdkConfiguration];
```

Reply:
this is not the config endpoint, sdk config is not returned.